### PR TITLE
Add acs1 test in tokenConverterTest

### PR DESCRIPTION
**Pull request by xiangxingxing**

### CODE REVIEW

File: test/AElf.Contracts.TokenConverter.Tests/Types/Round.cs [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using AElf.Types;
+using AElf.Sdk.CSharp;
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+
+namespace AElf.Contracts.Consensus.AEDPoS
+{
+    internal partial class Round
```
Comment (loning) [inline test/AElf.Contracts.TokenConverter.Tests/Types/Round.cs]: don't copy any code

File: test/AElf.Contracts.TokenConverter.Tests/Types/MinerList.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
+using System.Linq;
+using AElf.Sdk.CSharp;
+using AElf.Types;
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+
+namespace AElf.Contracts.Consensus.AEDPoS
+{
+    internal partial class MinerList
```
Comment (loning) [inline test/AElf.Contracts.TokenConverter.Tests/Types/MinerList.cs]: don't copy any code

File: test/AElf.Contracts.TokenConverter.Tests/Types/Round_Generation.cs [hunk 3]
```diff
@@ -0,0 +1,111 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using AElf.Sdk.CSharp;
+using Google.Protobuf.WellKnownTypes;
+
+namespace AElf.Contracts.Consensus.AEDPoS
+{
+    internal partial class Round
```
Comment (loning) [inline test/AElf.Contracts.TokenConverter.Tests/Types/Round_Generation.cs]: don't copy any code

File: test/AElf.Contracts.TokenConverter.Tests/TokenConverterTestBase.cs [hunk 4]
```diff
@@ -1,61 +1,105 @@
...
+using AElf.Kernel.Consensus;
 using AElf.Kernel.Consensus.AEDPoS;
 using AElf.Kernel.Token;
 using AElf.Types;
+using Volo.Abp.Threading;
 
 namespace AElf.Contracts.TokenConverter
 {
-    public class TokenConverterTestBase:ContractTestBase<TokenConverterTestModule>
+    public class TokenConverterTestBase : AEDPoSExtensionTestBase
```
Comment (rosona) [inline test/AElf.Contracts.TokenConverter.Tests/TokenConverterTestBase.cs]: why TokenConverterTestBase inherit  AEDPoSExtensionTestBase

**Pull request merged**